write-fonts/src/graph.rs

```diff
@@ -281,13 +312,47 @@ impl Graph {
                 //TODO: account for 'whence'
                 let rel_off = child.position - parent.position;
                 if link.len.max_value() < rel_off {
-                    result.push((*parent_id, link.object));
+                    result.push(Overflow {
+                        parent: *parent_id,
+                        child: link.object,
+                        distance: rel_off,
+                        offset_type: link.len,
+                    });
                 }
             }
         }
         result
     }
 
+    fn debug_overflows(&self, overflows: &[Overflow]) {
+        let (parents, children): (HashSet<_>, HashSet<_>) =
```

Comment:
if debug is not enabled early exit

Reply:
also should be behind log feature per other conversation? - np if that's out of scope here though.

Reply:
yea we don't have a log feature yet, will revisit that when I make that change (opened  #397 to track this)

Reply:
It looks like the early exit for log disabled - avoiding the iterations and zips to produce logs we never write - was not done?